javascript-data-model/main.js

```diff
@@ -0,0 +1,28 @@
+/* exported orderHistory */
+
+var orderHistory = [
+  {
+    itemNames: ['JavaScript for Impatient Programmers'],
+    type: ['book'],
```

Comment:
Rather than storing the item name and type in separate arrays, is there any way we can group together the data for each item in one piece of data? Something where each piece of data could potentially be individually named...